packages/venia-ui/lib/util/formValidators.js

```diff
@@ -159,3 +159,19 @@ export const isNotEqualToField = (value, values, fieldKey) => {
     };
     return value !== values[fieldKey] ? SUCCESS : message;
 };
+
+export const isValidateEmail = value => {
```

Comment:
Should be covered with a test sun you get approval from core member

Reply:
@larsroettig, Thank you! I have updated the test case for the email validation function.